static/src/javascripts/projects/common/modules/commercial/epic-utils.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+
+import fastdom from 'lib/fastdom-promise';
+import $ from 'lib/$';
+import { elementInView } from 'lib/element-inview';
+import reportError from 'lib/report-error';
+
+import { control as epicControlCopy } from 'common/modules/commercial/acquisitions-copy';
+import { epicButtonsTemplate } from 'common/modules/commercial/templates/acquisitions-epic-buttons';
+import { acquisitionsTestimonialBlockTemplate } from 'common/modules/commercial/templates/acquisitions-epic-testimonial-block';
+import { supportContributeURL } from 'common/modules/commercial/support-utilities';
+import { acquisitionsEpicControlTemplate } from 'common/modules/commercial/templates/acquisitions-epic-control';
+import {
+    addTrackingCodesToUrl,
+    submitInsertEvent,
+    submitViewEvent,
+} from 'common/modules/commercial/acquisitions-ophan';
+import { logView } from 'common/modules/commercial/acquisitions-view-log';
+
+import type { ReportedError } from 'lib/report-error';
+import type {
+    ABTestVariant,
+    ComponentEventWithoutAction,
+} from 'common/modules/commercial/acquisitions-ophan';
+
+export type EpicComponent = {
+    html: HTMLDivElement,
+    componentEvent?: ComponentEventWithoutAction,
+};
+
+const reportEpicError = (error: ReportedError): void => {
+    reportError(error, { feature: 'epic' }, false);
+};
+
+const controlEpicComponent = (abTest?: ABTestVariant): EpicComponent => {
+    const epicId = 'epic_control'; // TODO: check ok to use this
+    const epicComponentType = 'ACQUISITIONS_EPIC';
+    let testimonialBlock = '';
+    if (epicControlCopy.testimonial) {
+        testimonialBlock = acquisitionsTestimonialBlockTemplate(
+            epicControlCopy.testimonial
+        );
+    }
+    const rawEpic = acquisitionsEpicControlTemplate({
+        copy: epicControlCopy,
+        componentName: '', // TODO: confirm data-component not needed
+        buttonTemplate: epicButtonsTemplate({
+            supportUrl: addTrackingCodesToUrl({
+                base: supportContributeURL,
+                componentType: epicComponentType,
+                componentId: epicId,
+                campaignCode: epicId,
+                abTest,
+            }),
+        }),
+        testimonialBlock,
+        epicClass: '',
+        wrapperClass: '',
+    });
+    const epic = $.create(rawEpic).get(0);
+    return {
+        html: epic,
+        componentEvent: {
+            component: {
+                componentType: epicComponentType,
+                id: epicId,
+            },
+            abTest,
+        },
+    };
+};
+
+const insertEpic = (epic: HTMLDivElement): Promise<void> =>
```

Comment:
as discussed - maybe `insertAtSubmeta` would be a clearer name? and `content` instead of `epic`